specs/repository_name_changes.adoc

```diff
@@ -0,0 +1,151 @@
+// vim: tw=79
+
+= Repository name changes
+
+Repository names need to be updated to follow lowercase, hyphenated convention. The specification intend to cover affected repositories and any code and packaging changes required.
+
+Name of `Tendrl` github group itself is not affected by this change.
+
+
+== Problem description
+
+In Fedora, the RPM names usually match the upstream repository names. Currently for tendrl GitHub repositories are not aligned with rpm names. This would make it clearer to everyone that what code comes from where and help to maintain the overall packaging ecosystem consistency.
+
+== Use Cases
+
+None.
```

Comment:
My point was to mention about the intention behind the change in names. Its actually being done because rpm names should match the upstream project names. If thats correct, shouldn't we mention the same here?